crates/compiler/can/src/def.rs

```diff
@@ -2908,16 +2907,6 @@ pub struct IntroducedImport {
     exposed_symbols: Vec<(Symbol, Region)>,
 }
 
-impl IntroducedImport {
-    pub fn is_task(&self, env: &Env<'_>) -> bool {
-        // Temporarily needed for `!` convenience. Can be removed when Task becomes a builtin.
-        match env.qualified_module_ids.get_name(self.module_id) {
-            Some(name) => name.as_inner().as_str() == "Task",
-            None => false,
-        }
-    }
```

Comment:
wow lol this was a wild hack

Reply:
Yeah... just another thing that results from codegen. The more we have in the std lib, the fewer hacks like this one that we'll need.